install/streams.csv

```diff
@@ -2402,7 +2423,6 @@ WHZ,10,L,H,1,false,false,false,2001-04-07T21:53:00Z,9999-01-01T00:00:00Z
 WHZ,10,S,H,50,false,false,true,1993-03-22T15:45:00Z,2001-04-05T14:34:00Z
 WHZ,10,H,H,100,false,false,false,2001-04-07T21:53:00Z,9999-01-01T00:00:00Z
 WHZ,11,S,H,50,false,false,true,2001-04-08T10:20:00Z,2001-10-08T17:24:00Z
-WHZ,20,,,50,false,false,false,2002-03-02T00:00:00Z,2014-06-05T05:00:00Z
```

Comment:
From random checking, there are available data for BN(ENZ) streams
![WHZ_BN](https://user-images.githubusercontent.com/34523675/196128235-c2d62e9f-c707-4a49-b088-7466c13920c6.png)